com.unity.cinemachine/Runtime/Components/CinemachineRotationComposer.cs

```diff
@@ -187,7 +187,7 @@ public override void MutateCameraState(ref CameraState curState, float deltaTime
             {
                 if (deltaTime >= 0 && VirtualCamera.PreviousStateIsValid)
                     curState.RawOrientation = m_CameraOrientationPrevFrame;
-                return;  // navel-gazing, get outa here
```

Comment:
I think this is useful, `get outa here` may be not.

Reply:
please revert this change

Reply:
Not sure I understand what it means what is navel-gazing? Could we replace it with at least something more technically accurate?

Reply:
It's appropriate here.  Google it..

Reply:
Reverted it anyway it's a comment so it just concerns the few who will check the code